presets/advertising_indication.yml

```diff
@@ -0,0 +1,114 @@
+--- 
+author: PaDe
+description: 
+  default: "Pour participer à l'inventaire des préenseignes publicitaires en France, qui sont presque à 100% illégales."
+  en: "To participate in inventorying advertising indication boards in France, which are almost 100% unlawfull."
+  fr: "Pour participer à l'inventaire des préenseignes publicitaires en France, qui sont presque à 100% illégales."
+groups: 
+  - 
+    icon: "https://wiki.openstreetmap.org/wiki/File:Indication.svg"
+    items: 
+      - 
+        constraints: 
+          - 
+            action: 
+              key: "legal_status:FR"
+              type: SET_TAG_VALUE
+              value: illégal
+            condition: 
+              type: EQUALS
+              value: autres
+            source: 
+              key: message
+              type: TAG
+          - 
+            action: 
+              key: "legal_type:FR"
+              type: SET_TAG_VALUE
+              value: publicité
+            condition: 
+              type: EQUALS
+              value: autres
+            source: 
+              key: message
+              type: TAG
+        description: 
+          default: "Une préenseigne publicitaire, dispositif publicitaire hors-agglomération, pour présignaliser une activité."
+          en: "An indication board, found along a rural road, to advertise proximity of a craft/commercial/industrial activity. Rural advertising board."
```

Comment:
An indication board found along a rural road, which advertises the proximity of a craft/commercial/industrial activity.